tools/pipelines/templates/build-npm-package.yml

```diff
@@ -328,6 +328,51 @@ stages:
               mergeTestResults: false
             condition: and(succeededOrFailed(), eq(variables['startTest'], 'true'))
 
+          # Check Published Tests
+          - task: Bash@3
+            displayName: 'Check for nyc directory'
+            inputs:
+              targetType: 'inline'
+              workingDirectory: ${{ parameters.buildDirectory }}
+              script: |
+                test -d nyc && echo '##vso[task.setvariable variable=TestReportsExists;]true' || echo 'No nyc directory'
+            condition: and(succeededOrFailed(), eq(variables['startTest'], 'true'))
+
+          # Copy Test Reports
+          - task: CopyFiles@2
+            displayName: Copy Test Reports
+            inputs:
+              SourceFolder: ${{ parameters.buildDirectory }}/nyc
+              Contents: '**/*junit-report.xml'
+              TargetFolder: '$(Build.ArtifactStagingDirectory)/testFiles'
+            condition: and(succeededOrFailed(), eq(variables['TestReportsExists'], 'true'))
+
+          # Install parser library
+          - task: Bash@3
+            displayName: Install xml2js
+            inputs:
+              targetType: inline
+              script: |
+                npm i xml2js
```

Comment:
I am not enamored with just install it here without a lock file.  Could be unstable?

In general, I wonder if it is better if we use multi-mocha-reporter to emit both xml and json, so we don't need to parse XML?